CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Removed
+
+- Remove the limit of SpanEvents when exporting. (#2616)
```

Comment:
Not sure if it should not be in `Fixed` section instead

Reply:
Yeah, I think putting this in the `Fixed` section is better.

Reply:
done. Already changed. PTAL